### ISSUE TITLE
PHP - Class brackets highlighted wrong

### ISSUE THREAD

**Issue by nivv** [akamud/vscode-theme-onedark#69]
**Please provide as much information as possible, usually I need these information to actually start working on an issue, so try to fill as much of these as possible**

## Sample code

```php
<?php

namespace App\Repositories\TestClass;

class TestClass
{
    public function newFunction($world = 'world')
    {
        return 'hello' . ' ' . $world;
    }
}

```

## Filename and Language selected

`.php` | PHP

## Atom Original theme Screenshot

![skarmklipp 2017-07-21 08 40 12](https://user-images.githubusercontent.com/4033930/28451887-3e2d5584-6df0-11e7-9798-a9e024bcb3f6.png)

## VSCode theme Screenshot
![image](https://user-images.githubusercontent.com/4033930/28451895-4e8755c4-6df0-11e7-8d20-7e4542b634e3.png)

## Versions used

* VSCode version: Version 1.15.0-insider (1.15.0-insider)
* Theme version: 2.0.2

**Comment by trajedy** [comment]
Has become quite the OCD annoyance, also seeing the same issue with:

VSCode: 1.14.0
Theme: 2.0.2

**Comment by akamud** [comment]
Thanks for taking the time reporting this. I'll look into this for the next release.

**Comment by akamud** [comment]
This is fixed in 2.0.3.
